modules/scratchFetch.ts

```diff
@@ -0,0 +1,218 @@
+							thumbnail: {
+								url: ``,
+								height: 0,
+								width: 0,
+							},
+							author: {
+								name: `by: ${data.author.username}`,
+								url: `https://scratch.mit.edu/users/${data.author.username}`,
+								icon_url: data.author.profile.images["90x90"],
+							},
+							footer: {
+								text: notSet ? "Disable this using /settings" : "",
+								icon_url: ``,
+							},
+							url: `https://scratch.mit.edu/projects/${projectId}`,
+						},
+					],
+				});
+			})
+			.catch(() => {
+				//AKJGFDJHGADJHGJHADGJHGBDJKWD WHYYYYYYYYYYYYYYYYYYYYYY
+			});
+	} else if (type === "users") {
+		const username = urlParts[4];
+
+		const apiUrl = `https://api.scratch.mit.edu/users/${username}/`;
+
+		fetchApiData(apiUrl)
+			.then((data) => {
+				message.channel.send({
+					embeds: [
+						{
+							title: data.username,
+							description: data.scratchteam ? "Scratch Team" : "",
+							color: 0x00ffff,
+							fields: [
+								{
+									name: `WIWO`,
+									value: data.status,
+									inline: true,
+								},
+								{
+									name: `About`,
+									value: data.bio,
+									inline: true, //i tried to make this false but then it wouldnt send. instead of fixing the problem i ignored it
+								},
+							],
+							image: {
+								url: data.profile.images["90x90"],
+								height: 0,
+								width: 0,
+							},
+							thumbnail: {
+								url: ``,
+								height: 0,
+								width: 0,
+							},
+							author: {
+								name: ``,
+								url: ``,
+								icon_url: ``,
+							},
+							footer: {
+								text: notSet ? "Disable this using /settings" : "",
+								icon_url: ``,
+							},
+							url: `https://scratch.mit.edu/users/${username}`,
+						},
+					],
+				});
+			})
+			.catch(() => {});
+	} else if (type === "studios") {
+		const studioId = urlParts[4];
+
+		const apiUrl = `https://api.scratch.mit.edu/studios/${studioId}/`;
+
+		fetchApiData(apiUrl)
+			.then((data) => {
+				//mnm,nmnm,nn/m.n.,nmn/mn
+
+				message.channel.send({
+					embeds: [
+						{
+							title: data.title,
+							description: long(data.description, 400, "..."),
+							color: 0x00ffff,
+							fields: [
+								{
+									name: `comments`,
+									value: data.comments_allowed
+										? `${data.stats.comments}`
+										: "comments off",
+									inline: true,
+								},
+								{
+									name: `followers`,
+									value: data.stats.followers,
+									inline: true,
+								},
+								{
+									name: `managers`,
+									value: data.stats.managers,
+									inline: true,
+								},
+								{
+									name: `projects`,
+									value: data.stats.projects,
+									inline: true,
+								},
+							],
+							image: {
+								url: data.image,
+								height: 0,
+								width: 0,
+							},
+							thumbnail: {
+								url: ``,
+								height: 0,
+								width: 0,
+							},
+							author: {
+								name: ``,
+								url: ``,
+								icon_url: ``,
+							},
```

Comment:
remove

unfortunately the only thing we could put here is the host's icon, and the name is required so that don't work